test/TestProjects/MgmtTest-TypeSpec/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Release History
```

Comment:
Try namespace not start with "Microsoft" or "Azure", that might not generate this file.

Reply:
Updated